DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -301,5 +301,24 @@ public void SetCentreAutoRegistered_should_set_AutoRegistered_true()
                 transaction.Dispose();
             }
         }
+
+        [Test]
+        public void GetAllCentreSummaries_returns_all_summary_details_and_reference_data()
+        {
+            using var transaction = new TransactionScope();
+            try
+            {
+                // When
+                var summaries = centresDataService.GetAllCentreSummaries();
+
+                // Then
+                summaries.Should()
+                    .OnlyContain(c => c.CentreType != null && c.RegionName != null && (c.Active || !c.Active));

```

Comment:
I'm a bit confused as to what this line is testing. Centres are either Active or Inactive so that part is always true. The CentreType and RegionName property check seems off to me as well - this seems to just be checking that data in the database exists. Does that make sense or am I just confused?

I would expect that we would have some kind of count check to make sure that all centres are returned.

Reply:
I've had some question marks over these data services tests and the reliance on a "static" database for testing.
For now I've changed the test to be very specific. Although I have concerns given the reliance on the test data always being the same.

Reply:
That's a good point. We have been (mostly) relying on the assumption of a "static" database throughout the data tests, and I believe that the assumption is a pretty safe one to make.

As far as I am aware, there are not any plans to majorly change a bunch of data, and seeing as we use a separate database only for unit tests (that should be identical to the back up file) then it should be pretty reliable. We've only recently moved to a preference of setting up data in a unit test, and the docs are pretty clear about transaction usage. So overall, I think we can rely on the data being consistent. If you have any questions/concerns, feel free to ask.